app/views/content/shared/block-promos/_adviser_findsalary.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::AdviserBlockComponent.new %>
+      <%= render CallsToAction::FindBlockComponent.new(
+        link_target: "https://find-teacher-training-courses.service.gov.uk/results?funding%5B%5D=salary&funding%5B%5D=apprenticeship&applications_open=true&subject_name=&subject_code=&location=&radius=10&provider_name=&provider_code=&order=course_name_ascending"
```

Comment:
We are not given any indication that we are sending them to a pre-filtered search result - shouldn't we? Given that we are updating the link, shouldn't we update the promo body text as well?